pkg/e2e/verify/pod_webhook.go

```diff
@@ -0,0 +1,209 @@
+package verify
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/rest"
+)
+
+var podWebhookTestName string = "[Suite: e2e] [OSD] pod validating webhook"
+
+func init() {
+	alert.RegisterGinkgoAlert(podWebhookTestName, "SD-SREP", "Alice Hubenko", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
+}
+
+var _ = ginkgo.Describe(podWebhookTestName, func() {
+
+	h := helper.New()
+
+	ginkgo.Context("pod webhook", func() {
```

Comment:
Is there a reason you specify multiple ginkgo Contexts? I think you should only use one and put all the `It` blocks underneath it

Reply:
I used it for running individual tests. I updated so that there's a single Context.